### PR TITLE
[Win32] Create JS Bundle for E2E Spec Files and Publish as NPM Package

### PR DESCRIPTION
**Pull request by samuelfreiberg**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [X] win32 (Office)
- [X] windows
- [ ] android

### Description of changes

For native testing, we want a JS bundle of all the spec files to execute. This creates a JS bundle using webpack and exports it as part of the `@fluentui-react-native/e2e-testing-specs` NPM package

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by dannyvv — APPROVED** [review]

File: apps/fluent-tester/src/E2E/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@fluentui-react-native/e2e-testing-specs",
```
Comment (Saadnajmi) [inline apps/fluent-tester/src/E2E/package.json:2]: So we have a package inside a package? Feels like that might break something somewhere 😅

File: apps/fluent-tester/src/E2E/webpack.config.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import * as path from 'path';
+import { Configuration } from 'webpack';
```
Comment (Saadnajmi) [inline apps/fluent-tester/src/E2E/webpack.config.ts:2]: Why web pack and not metro?

**Review by Saadnajmi — APPROVED** [review]

File: apps/fluent-tester/src/E2E/webpack.config.ts (hunk 2 above)
Comment (rurikoaraki) [reply apps/fluent-tester/src/E2E/webpack.config.ts:2]: +1, we're generally moving away from webpack.

Comment (samuelfreiberg) [reply apps/fluent-tester/src/E2E/webpack.config.ts:2]: Wasn't aware of moving away from webpack. Worked with Webpack more in the OMR, but I'll change to Metro. Nbd - thanks for letting me know

File: apps/fluent-tester/src/E2E/package.json (hunk 1 above)
Comment (samuelfreiberg) [reply apps/fluent-tester/src/E2E/package.json]: Files changed 9->144 :D

Comment (Saadnajmi) [reply apps/fluent-tester/src/E2E/package.json]: 😅 I trust your judgement either way. I'm not sure what is better TBH

File: apps/E2E/src/FocusZone/pages/FocusZonePageObject.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { FocusZoneDirection } from '@fluentui/react-native';
+import { FocusZoneDirection } from '@fluentui-react-native/focus-zone';
```
Comment (Saadnajmi) [inline apps/E2E/src/FocusZone/pages/FocusZonePageObject.ts:1]: Fun: D

File: apps/fluent-tester/src/E2E/webpack.config.ts (hunk 2 above)
Comment (samuelfreiberg) [reply apps/fluent-tester/src/E2E/webpack.config.ts]: Changed to Metro, resolving

**Pull request merged**